subsystem-test/framework/src/main/java/org/jboss/as/subsystem/test/SubsystemTestDelegate.java

```diff
@@ -584,8 +584,8 @@ public KernelServices build() throws Exception {
                         //we check the reject policy simulating a reject before adding it to the list of boot operations
                         if (!transformedOp.rejectOperation(SUCCESS)) {
                             transformedBootOperations.add(transformedOp.getTransformedOperation());
-//                        } else {
-//                            System.out.println(transformedOp.getFailureDescription());
+                        } else {
+                            System.out.println(transformedOp.getFailureDescription());
```

Comment:
@jasondlee I don't think this should be uncommented.

Reply:
You're right. I had a random test failure that I was trying to figure out, but it seems to have worked itself out and I forgot about that. Change incoming.